src/components/shared/Entity/DetailsForm.tsx

```diff
@@ -145,7 +145,7 @@ function DetailsForm({
                 elements: [
                     {
                         label: intl.formatMessage({
-                            id: 'connector.label',
+                            id: 'entityCreate.connector.label',
```

Comment:
Switched label from a common one to one that adds "search" to it

Reply:
Curious if placeholders in (certain) form fields should be a short statement that begins with a verb. For example, an active GitHub comment text field without text uses _Leave a comment_ as its placeholder text. Wondering if a similar approach would be helpful here (e.g., _Select a connector_). Thinking out loud.

Reply:
For this field we specifically want to let them know they can search/type in the connector name.